selfdrive/car/nissan/interface.py

```diff
@@ -47,7 +47,7 @@ def _update(self, c):
     be.type = car.CarState.ButtonEvent.Type.accelCruise
     buttonEvents.append(be)
 
-    events = self.create_common_events(ret)
+    events = self.create_common_events(ret, extra_gears=[car.CarState.GearShifter.brake])
```

Comment:
Checked and this mode is only in the leaf DBC, so we can be sure that it will never occur on other cars